src/functionaltests/java/com/ericsson/ei/utils/DataBaseManager.java

```diff
@@ -172,10 +171,16 @@ public String getValueFromQuery(List<String> databaseQueryResult, String key, in
      * @return int of the size of the waitlist.
      */
     public int waitListSize() {
-        mongoClient = new MongoClient(getMongoDbHost(), getMongoDbPort());
-        MongoDatabase db = mongoClient.getDatabase(database);
-        MongoCollection<Document> table = db.getCollection(waitlistCollectionName);
+        MongoCollection<Document> table = getCollection(waitlistCollectionName);
```

Comment:
Maybe either call it getTable or change the name table to collection? Collection is maybe the better name as that is what's being used in mongodb

Reply:
Done in next commit.